lib/models/collected_item_data.dart

```diff
@@ -0,0 +1,281 @@
+class CollectionItem {
+  final String imageName;
+  final String categoryName;
+  final bool isEnabled;
```

Comment:
isEnabled should come from the UserProfile. This should only have static information on the collection